Entities/Items/Crate/Crate.as

```diff
@@ -162,6 +185,36 @@ bool doesCollideWithBlob(CBlob@ this, CBlob@ blob)
 	return !blob.hasTag("parachute");
 }
 
+void onCollision( CBlob@ this, CBlob@ blob, bool solid ){
+	
+	if(!isServer())return;
+	
+	if(blob !is null){
+		if (!blob.isAttached() && blob.isOnGround() && blob.hasTag("material") && blob.getName() != "mat_arrows")
+		{
+			this.server_PutInInventory(blob);
+		}
+	}
+	
+	if(!solid || this.getOldVelocity().Length() < 6.0f)return; 
+	
+	CInventory@ inv = this.getInventory();
+	
+	if(inv !is null){
+		for(int i = 0;i < inv.getItemsCount();i++){
+			CBlob @item = inv.getItem(i);
+			if(item !is null){
+				if(item.hasTag("player")){
+					//Enable this line if you want crates to damage players on impact
+					//this.server_Hit(item,item.getPosition(),Vec2f(0,0),1.0f,Hitters::fall,true);
```

Comment:
Sounds like you could pass the velocity in there, so the fall damage applies accordingly.

Reply:
Ah yeah, I was thinking about it at first, but since knight can literally just fly with crates, applying damage/forcing a fall just makes a solo crates from catapults much worse/useless. (and the idea behind only dealing 1 damage is so that any class can survive the fall)